### ISSUE TITLE
Validation Guide: Links Between PMMA Material Used in NIST Parallel Panel and Polymer Cases

### ISSUE THREAD

**Issue by TristanHehnen** [firemodels/fds#11922]
Hi,

to my understanding, the PMMA material used for the NIST parallel panel simulation is also used for the NIST Polymer cases in the FDS validation suite. 
Could this please made explicit in the validation guide and could the respective text sections reference each other?

Best,
Tristan

**Comment by TristanHehnen** [comment]
Also, in figure 14.8 in the nightly built validation guide the MCC data is a bit off, while the TGA data fits quite well.
We saw similar behaviour when trying to match MCC and TGA data for MaCFP PMMA. It was reported for example in [Ding et al.](https://www.sciencedirect.com/science/article/abs/pii/S0010218019300136?via%3Dihub) that they found similar behaviour and that they shifted the MCC data a bit to higher temperatures. 
If something similar was performed in figure 14.8, please point this out in the text.

**Comment by mcgratta** [comment]
Yes, I will update the various sections of the Validation Guide to reflect what we are currently doing with PMMA.

**Comment by TristanHehnen** [comment]
Thank you very much!

**Comment by mcgratta** [comment]
I added some cross-links to the document to connect experiments that are part of the MaCFP. As far as Fig 14.8, I'll ask Jason Floyd to take a look because he is doing the MaCFP Gasification Apparatus example.

**Comment by drjfloyd** [comment]
Based on the recent discussion in the [MaCFP Group](https://groups.google.com/g/macfp-discussions/c/S61GPR4ObjY) I don't know that we should expect these to line up for the MCC test. I think this figure might be better as a gassification test figure.
